handler.py

```diff
@@ -273,7 +205,7 @@ def get_data(event, context):
         payload["messages"] = messages
     return dict(
         headers={"Content-Type": "application/json"},
-        status=200,
+        statusCode=200,
```

Comment:
another one of these needs fixing: "Content missing or required" in get_data

Reply:
crap, thanks!